app.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
We will have to get your heroku PR builds setup too. @mhemesath Ideally you or someone on your team would setup the App so we can get the most out of our free accounts.